### DISCUSSION TITLE
Angular support

### DISCUSSION THREAD

**Discussion by thiyaga05** [plouc/nivo#2007] [General]
1. Does this nivo npm supports for the angular project?
2. How to implement the types of nivo charts inside the angular? can you give some sample ?

**Comment by plouc** [comment]
This project doesn't support Angular, implementing something similar in Angular would require quite some work, I can definitely not provide an example for this 😅.

**Reply by thiyaga05** [reply]
Does the implementation of react inside the angular project, will help to use the nivo packages?

**Reply by plouc** [reply]
@thiyaga05, it's hard to answer, I have no experience with such setup (and I would personally not recommend doing that), you would also end up with a lot of dependencies mixing Angular and React. I'd suggest that you find a similar library for Angular instead, it would be a safer option IMHO.
